batch/gcp-create-worker-image.sh

```diff
@@ -18,11 +18,7 @@ create_build_image_instance() {
     python3 ../ci/jinja2_render.py '{"global":{"docker_root_image":"'${DOCKER_ROOT_IMAGE}'"}}' \
         build-batch-worker-image-startup-gcp.sh build-batch-worker-image-startup-gcp.sh.out
 
-    UBUNTU_IMAGE=$(gcloud compute images list \
-        --standard-images \
-        --filter 'family="ubuntu-minimal-2004-lts"' \
-        --format='value(name)')
-
+    UBUNTU_IMAGE=ubuntu-minimal-2004-focal-v20230216
```

Comment:
what's the thinking on this change?

Reply:
They added another image that made this query return two images and not one, which broke the invocation. More broadly, this is in the spirit of pinning our dependencies.